assignment.md

```diff
@@ -34,30 +34,47 @@ Once you're finished with your edits, send the PR link to the HashiCorp recruite
 ---
 
 OVERALL QUESTIONS
-1. 
 2. 
 3. 
 
 ## Content
 -->
 
 <!-- EAJ TODO: Determine best structure for the document after working with the material. Ideas: Subheadings for each command; ol that contains an extended explanation; table that keeps command, description, and why to use it in rows -->
+# Managing code via the command line
+Git offers you many ways to manage your code both locally and remotely. Create branches to separate updates from production code, create patches, test ideas, and merge your changes into the codebase. The command line makes it easy to download, review, modify, and push changes between your local machine and a remote repository. <!-- New audiences may require a more in-depth intro, so we're leaving room for that here. -->
 
-### Understanding `push`, `pull`, and `fetch`
+<!-- Thought again; we could either link off to docs around local/remote branches and repositories if users need a refresh on this documentation or if they're newer to git and would require this information. -->
 
-<!-- EAJ TODO: Need intro that explains purpose. Why are we documenting this? Why is it important? -->
+## Git code management commands
 
-- `git fetch` - get changes from a remote repo into your tracking branch
-- `git pull` - will get changes from a remote branch into your tracking branch and merge them into a local branch
-- `git push` - sent changes from a local branch to a remote repo
+<!-- EAJ NOTE: Rearranging because the foundations build on each other -->
+
+<!-- EAJ NOTE: I am removing "git" from the code phrases below. I am assuming that this is part of a larger docset that has already introduced the concept of git on the command line and we've moved on to tasks users will need to complete. -->
+Git includes three commands that you can use to review, update, and publish your changes to and from your local and remote repositories.
+- `fetch` - Retrieve code updates from a remote repository to place into your tracking branch.
```

Comment:
Might want to add `git` back into these; convention from external docs seems to use it more often than not.